docs/experimental/bicep-mcp-server.md

```diff
@@ -0,0 +1,46 @@
+# Using Bicep MCP Server in VS Code (Preview!)
```

Comment:
You'll need to also update https://github.com/Azure/bicep/blob/main/docs/experimental-features.md to link to this doc